pkg/controller/cincinnati/new.go

```diff
@@ -62,6 +63,31 @@ data_directory = "/tmp/cincinnati/graph-data"
 [[plugin_settings]]
 name = "edge-add-remove"`
 
+// TOML when graph data source is an init container
+const graphBuilderTOMLInitContainer string = `verbosity = "vvv"
```

Comment:
If we can, I believe we should have one template to rule them all. Tinkering around with this for a few minutes and I came up with https://play.golang.org/p/eHIbv22xAeI WDYT?